### DISCUSSION TITLE
#Issue - To bring 2d data in model for cross-probing.

### DISCUSSION THREAD

**Discussion by RameshwarShinde** [iTwin/itwinjs-core#3646] [Q&A]
Hi,

Implementing cross-probing with reference of your given CrossProbing sample.
Facing issue to get the 2d data in schema for particular model. Trying this with Revit's (.rvt) files and object is getting empty for 2d data. i.e. no data in BisCore.GeometricModel2d.

So, question is :-
1.  Is there any way to add/ append 2d data to the model I'm using? If yes then How?
2.  Is there any criteria in iTwin for the model to load? like specific type of file only we should load OR model should not be .rvt file etc.

Plz help for such.

**Comment by pmconne** [comment]
Go to [iModelConsole](https://imodelconsole.bentley.com).
Open your iModel.
Enter query `SELECT * FROM bis.ViewDefinition2d`
Any results?

**Reply by RameshwarShinde** [reply]
Exactly , that's the scenario. It's getting empty. 'No results found'. @pmconne
